omod/src/main/webapp/resources/htmlforms/055b-HCTClientCard.xml

```diff
@@ -10,227 +10,339 @@
+                  return false;
+               }
+               if(! jq("#162925").find("select").is(':disabled') &amp;&amp; jq("#162925").find("select").find(":selected").val()== 90002 &amp;&amp; jq("#99115").find("input[type$='text']").val()=='') {
+                   getField('99115.error').html('Required text field is empty').show();
+                   return false;
+               }
+               if(! jq("#165160").find("select").is(':disabled') &amp;&amp; jq("#165160").find("select").find(":selected").val()== 90002 &amp;&amp; jq("#165159").find("input[type$='text']").val()=='') {
+                    getField('165159.error').html('Required text field is empty').show();
+                    return false;
+               }
+               if(! jq("#165168").find("select").is(':disabled') &amp;&amp; jq("#165168").find("select").find(":selected").val()== 90002 &amp;&amp; jq("#165167").find("input[type$='text']").val()=='') {
+                    getField('165167.error').html('Required text field is empty').show();
+                    return false;
+               }
+
+                return true;
+            });
+        }
+
+    </script>
+
+
+    <div class="ai-page-frame" id="form-id">
         <div class="headers" style="text-align: center;">
             <h3 style="background: #FFFF7D; padding: 10px;">HCT Client Card</h3>
         </div>
         <section>
             <h3>Section A:</h3>
             <table>
                 <tbody>
-                    <tr>
-                        <td>
-                            <label>Date:</label>
-                            <encounterDate default="today" disallowMultipleEncountersOnDate="block"/>
-                            <span style="display: none;">
+                <tr>
+                    <td>
+                        <label>Date:</label>
+                        <encounterDate default="today" disallowMultipleEncountersOnDate="block"/>
+                        <span style="display: none;">
                                 <encounterLocation default="629d78e9-93e5-43b0-ad8a-48313fd99117"
                                                    order="629d78e9-93e5-43b0-ad8a-48313fd99117"/>
                             </span>
-                        </td>
-                        <td>
-                            <label>Serial No:</label>
-                            <obs conceptId="164985" required="true"/>
-                        </td>
-                        <td>
-                            <label>National ID:</label>
-                            <patient field="identifier" identifierTypeId="f0c16a6d-dc5f-4118-a803-616d0075d282"
-                                     required="false"/>
-                        </td>
-                    </tr>
+                    </td>
+                    <td>
+                        <label>Serial No:</label>
+                        <obs conceptId="1646" required="true"/>
+                    </td>
+                    <td>
```

Comment:
@dbaluku Add validation rule for National ID from aijar - its a regex that can be applied

Reply:
@dbaluku as discussed remove this field from here we shall validate it from the registration form

Reply:
Done